examples/resources/tsb_user/resource.tf

```diff
@@ -0,0 +1,9 @@
+resource "tsb_user" "example" {
+  organization = "tetrate"
```

Comment:
since we're mostly single-org today, and an org will be hard tenancy, would it make sense to consider this a "provider" setting so you dont' have to set it in every single resource?

Reply:
Or even better, why don't just have a `parent` field instead of org, that is an opaque id (the fqn) so that we save all the trouble parsing FQNs, that we shouldn't be doing anyway?

Reply:
Keeping the org here is fine.

But (agree to do this in another PR) let's not split the FQNs and treat them as opaque IDs in the TF provider. There will be other resources that will reference resources by FQN (policy bindings, for example), and we want to be consistent.

Also, fqns should be opaque ids and never be aprsed anyway, so let's make sure we do that, and in a future PR change this `organization` field by a `parent` field.